Modules/ST-SEGMENT/st-segment.h

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <vector>
+#include <complex>
+#include <iostream>
+#include <valarray>
+#include <cmath>
+#include <numeric>
+#include <cstdlib>
+#include <algorithm>
+#include <memory>
+#include <string>
+
+
+using namespace std;
+class STsegment{
+public:
+    STsegment();
+
```

Comment:
Spróbujcie zastosować prywatne pola zachowujące wyniki działania poszczególnych etapów modułu. Pamiętajcie także proszę o stosowaniu `std::move()` jest to funkcja dosłownie przenosząca wektory w inne miejsce (likwidujemy kopiowanie które jest mega nie optymalną praktyką) Z grubsza - KOPIOWNIA == TWÓJ WRÓG NR 1